src/utils/mouseData.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Deze file snap ik niet zo goed

Reply:
> Deze file snap ik niet zo goed

Omdat we geen firestore kunnen gebruiken in de preload scripts, stuur ik vanuit de mainwindow de muisdata naar firestore